client/tests/integration/manuscript-manager-template-test.coffee

```diff
@@ -23,7 +23,10 @@ module 'Integration: Manuscript Manager Templates',
   setup: ->
     app = startApp()
     server = setupMockServer()
-
+    Tahi.__container__.lookup(
+      'controller:admin/journal/manuscript-manager-template/edit'
+    )._actions.saveTemplateOnClick = -> console.log 'No Action'
+      
```

Comment:
To address that, can we end each test with either a save or a cancel click? I'd like to avoid this stubbing if possible.